server/server.go

```diff
@@ -177,6 +177,8 @@ func (s *Server) handle(conn *gokeyless.Conn) {
 		case gokeyless.OpECDSASignSHA1:
 			opts = crypto.SHA1
 		case gokeyless.OpRSASignSHA224:
+			isRSA = true
```

Comment:
Not a bad idea (I had actually forgotten this was valid syntax). This use actually eliminates the need for the `isRSA` flag altogether.